### ISSUE TITLE
TOTP

### ISSUE THREAD

**Issue by q16marvin** [schorschii/webpw#5]
Hi,

do you have plans to add "totp authenticator" on your roadmap? we are currently thinking about create own "password safe" with ldap user backend and totp :)

Thx!

**Comment by schorschii** [comment]
Yes, this would be a nice addition in the future, but first I want to release v1.0 which currently waits for an external code audit. After that I can imagine implementing this for v1.1. I leave this issue open until then.
